components/dashboard/src/repositories/detail/RemoveConfiguration.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { Heading2, Subheading } from "@podkit/typography/Headings";
+import { Button } from "../../components/Button";
+import { RemoveConfigurationModal } from "./RemoveConfigurationModal";
+import { useHistory } from "react-router";
+import { useCallback, useState } from "react";
+import type { Configuration } from "@gitpod/public-api/lib/gitpod/v1/configuration_pb";
+import { ConfigurationSettingsField } from "./ConfigurationSettingsField";
+
+interface Props {
+    configuration: Configuration;
+}
+
+export const RemoveConfiguration = ({ configuration }: Props) => {
+    const [showRemoveModal, setShowRemoveModal] = useState(false);
+
+    const history = useHistory();
+    const onProjectRemoved = useCallback(() => {
+        history.push("/projects");
```

Comment:
This should go to `/repositories` I think, right?

Reply:
You're super-duper correct